lib/worker/subprocess.js

```diff
@@ -41,12 +42,26 @@ ipc.options.then(async options => {
 		process.exit(); // eslint-disable-line unicorn/no-process-exit
 	}
 
+	// TODO: Initialize providers here, then pass to lineNumberSelection() so they
+	// can be used to parse the test file.
+	let checkSelectedByLineNumbers;
+	try {
+		checkSelectedByLineNumbers = lineNumberSelection({
+			file: options.file,
+			lineNumbers: options.lineNumbers
+		});
+	} catch (error) {
+		ipc.send({type: 'line-number-selection-error', err: serializeError('Line number selection error', false, error, options.file)});
+		checkSelectedByLineNumbers = () => false;
+	}
+
 	const runner = new Runner({
 		experiments: options.experiments,
 		failFast: options.failFast,
 		failWithoutAssertions: options.failWithoutAssertions,
 		file: options.file,
 		match: options.match,
+		checkSelectedByLineNumbers,
```

Comment:
Alphabetical ordering? If not, move shorthand property to bottom?

Reply:
Good spot, thanks.